config/routes.rb

```diff
@@ -12,13 +12,14 @@
   get 'meta_guide_to_upgrading_ember', to: 'pages#meta_guide_to_upgrading_ember'
 
   get 'account', to: 'accounts#show'
+  get 'sales', to: 'accounts#sales'
   post 'subscribe', to: 'accounts#subscribe'
   post 'unsubscribe', to: 'accounts#unsubscribe'
   post 'change_credit_card', to: 'accounts#change_credit_card'
   post 'reset_password_through_email', to: 'accounts#reset_password_through_email'
 
   get 'set_password/:user_id/:token', to: 'users#set_password'
-  post 'set_password/:user_id/:token', to: 'users#activate_organization_user'
+  post 'set_password/:user_id/:token', to: 'users#açctivate_organization_user'
```

Comment:
this would have been bad to let through.  Fixing.  Another reason to always review your own changes file by file (in something like gitx) before committing.